infrastructure/preview.tfvars

```diff
@@ -1,3 +1,7 @@
+idam_authentication_web_url = "https://idam.preprod.ccidam.reform.hmcts.net"
```

Comment:
Why do we have `idam_authentication_web_url` and `authentication_web_url`? I believe one of them is not used and should be removed

Reply:
authentication_web_url is now removed.